crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -1823,7 +1823,11 @@ impl OlmMachine {
         from_backup: bool,
         progress_listener: impl Fn(usize, usize),
     ) -> StoreResult<RoomKeyImportResult> {
-        self.store().import_room_keys(exported_keys, from_backup, progress_listener).await
+        let backup_version =
```

Comment:
I think we can just remove this method. We had a `0.7.x` release.

Reply:
Even better! Will remove it.